dist/css/boosted.css

```diff
@@ -2132,7 +2132,7 @@ h2, .h2,
   box-shadow: inset 0 0 0 9999px var(--bs-table-accent-bg);
 }
 .table > thead > tr {
-  border-bottom-width: calc(var(--bs-border-width) * 0.5 * 2);
+  border-bottom-width: calc(calc(var(--bs-border-width) * 0.5) * 2);
```

Comment:
We have double calcs here, don't know if this is mandatory or not

Reply:
Comes from `calc(#{$table-border-width} * 2)`, I'd say it's mandatory in the usage in `_tables.scss` because you can't know whether `$table-border-width` is already a `calc` or not. But in the end CSS file, agree that it's useless. IDK if it can be simplified by a plugin, tool, etc. 🤷